django/db/models/expressions.py

```diff
@@ -803,6 +803,22 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
         return super().resolve_expression(query, allow_joins, reuse, summarize, for_save)
 
 
+class Default(BaseExpression):
+    """Expression for ``DEFAULT`` clause in ``INSERT`` or ``UPDATE`` query."""
+
+    def __repr__(self):
+        return 'DEFAULT'
+
+    def as_sql(self, compiler, connection):
+        return 'DEFAULT', []
+
+    def as_sqlite(self, compiler, connection):
+        return 'NULL', []
+
+    def as_oracle(self, compiler, connection):
+        return 'NULL', []
```

Comment:
I think these will do the wrong thing if the column is nullable and has a default, unless SQLite and Oracle both treat the presence of a `NULL` as an instruction to use the default value. If that is the case, it's worth a comment to document the behaviour.